packages/host/app/components/ai-assistant/chat-input/index.gts

```diff
@@ -0,0 +1,103 @@
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import onKeyMod from 'ember-keyboard/modifiers/on-key';
+
+import { BoxelInput, IconButton } from '@cardstack/boxel-ui/components';
+import { Send } from '@cardstack/boxel-ui/icons';
+import { setCssVar } from '@cardstack/boxel-ui/modifiers';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    value: string;
+    onInput: (val: string) => void;
+    onSend: (val: string) => void;
+  };
+}
+
+// @keyResponder
+export default class AiAssistantChatInput extends Component<Signature> {
+  <template>
+    <div class='chat-input-container'>
+      <label for='ai-chat-input' class='boxel-sr-only'>
+        Enter text to chat with AI Assistant
+      </label>
+      <BoxelInput
+        class='chat-input'
+        @id='ai-chat-input'
+        @type='textarea'
+        @value={{@value}}
+        @onInput={{@onInput}}
+        @placeholder='Enter text here'
+        {{onKeyMod 'cmd+Enter' this.onSend}}
+        {{onKeyMod 'ctrl+Enter' this.onSend}}
+        {{setCssVar chat-input-height=this.height}}
+        ...attributes
+      />
+      <IconButton
+        {{on 'click' this.onSend}}
+        class='send-button'
+        @icon={{Send}}
+        @height='20'
+        @width='20'
+      >
+        Send
+      </IconButton>
+    </div>
+    <style>
+      .chat-input-container {
+        display: grid;
+        grid-template-columns: 1fr auto;
+        padding: var(--boxel-sp-xs);
+        background-color: var(--boxel-light);
+      }
+      .chat-input {
+        height: var(--chat-input-height);
+        border-color: transparent;
+        font-weight: 500;
+        padding: var(--boxel-sp-xxs);
+        resize: none;
+      }
+      .chat-input::placeholder {
+        color: var(--boxel-400);
+      }
+      .chat-input:hover:not(:disabled) {
+        border-color: var(--boxel-border-color);
+      }
+      .send-button {
+        --icon-color: var(--boxel-highlight);
+        height: 30px;
+        padding-bottom: var(--boxel-sp-xs);
+      }
+      .send-button:hover:not(:disabled) {
+        --icon-color: var(--boxel-highlight-hover);
+      }
+      .send-button > :deep(svg) {
+        margin: 0;
```

Comment:
`IconButton` [overrides this](https://github.com/cardstack/boxel/blob/121186e11093fe057c15ec533f93ab99146abee8/packages/boxel-ui/addon/src/components/icon-button/index.gts#L46) so it seems unnecessary?

<img width="214" alt="Host Components 2024-01-09 13-01-41" src="https://github.com/cardstack/boxel/assets/43280/e8793f12-6006-4acd-83d6-6756f1fbddf4">

Reply:
yes, meant to remove that as well